homework/tests/example.spec.js

```diff
@@ -1,15 +1,1516 @@
 // @ts-check
 const { test, expect } = require('@playwright/test');
 
-test.describe('Concatenate', () => {
-  test('Concatenating 2 and 3 results in 23', async ({ page }) => {
-    await page.goto('https://testsheepnz.github.io/BasicCalculator');
-    await page.selectOption('#selectBuild', { label: 'Prototype'});
-    await page.locator('#number1Field').type('2');
-    await page.locator('#number2Field').type('3');
-    await page.selectOption('#selectOperationDropdown', {label: 'Concatenate'});
-    await page.locator('#calculateButton').click();
-
-    await expect(page.locator('#numberAnswerField')).toHaveValue('24');
+const data = [
+  'Prototype',
```

Comment:
"Prototype version has no known flaws" So let's assume that prototype is without bugs and works as it should work. And let modify tests to make it all pass or remove failing. 
Yes there are abnormal counting but it's a java script

Reply:
I removed "Prototype" version and made 9 build in separate tests, because it's inappropriate to test 9 build functionality when main elements of 9 build aren't visible. I commented tests that get the same bugs, now are less of them.